proposal.md

```diff
@@ -461,22 +461,53 @@ their package evolves. Note that CRAN permits dependencies on Bioconductor
 
 `tools::check_packages_in_dir` can be used to check the reverse dependencies of a package (or set of packages).
 
-CONSIDER: checked, `sessioninfo::session_info`, revdepcheck, prefixer, rcheology, tools like drat and miniCRAN
-
 WRE reference: [Package Dependencies](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Package-Dependencies).
 
-
-
-SEE ALSO:  
-<https://github.com/ropensci-archive/PackageDevelopment/blob/master/README-NOT.md#dependency-management>, https://github.com/IndrajeetPatil/awesome-r-pkgtools?tab=readme-ov-file#dependency-management-%EF%B8%8F
+ - `r pkg("attachment")` provides helpers to update forward dependencies in your 
+ DESCRIPTION as required by changes to `.R` or `.Rmd` files, and to quickly 
+ install missing packages declared in a DESCRIPTION file.
+ - `r pkg("rcheology")` provides a dataset of functions in all base and 
+ recommended packages of R from version 0.50. This, or its companion 
+ [rcheology Shiny app](https://hughjonesd.shinyapps.io/rcheology/) can help 
+ to determine the minimum version of R on which a package depends.
+ - `r pkg("backports")` provides reimplementations of functions introduced or 
+ changed since R v3.0.0. This enables package developers to maintain 
+ compatability with older versions of R when using newer functionality.
+ - `r pkg("sessioninfo")` provides `session_info()` as an alternative to 
+ `utils::sessionInfo`, which returns richer information on add-on packages. 
+ - `r pkg("pacs")` provides various utilities for managing packages, including 
+ `pac_timemachine()` to get the package version at a certain date, functions to 
+ compare the DESCRIPTION or NAMESPACE across versions, and `pac_deps_heavy()` 
+ to identify how heavy a package's forward dependencies are in terms of the 
+ number of forward dependencies they have. 
+ - `r pkg("remotes")` provides `install_version()` to install a particular 
+ version of a package, while `r pkg("dateback")` can be used to install 
+ packages based on a date or date range.
+ - `r pkg("pkgndep")` provides tools to assess "dependency heaviness" (the 
+ number of forward dependencies added by depending on a new package) and 
+ provides suggestions for optimizing package dependencies.
+ - `r pkg("pkgdepends")` can be used to identify, visualise and install package 
+ dependencies, including those specified via `Remotes` in the DESCRIPTION, 
+ for packages on CRAN, Bioconductor, and git repositories.
+ - `r pkg("pkggraph")`, `r pkg("pkgnet)`, `r pkg("deepdep")`, `r pkg("crandep")` 
+ and `r pkg("cranly")` provide  functionality to visualise package dependencies.
+ - `r pkg("pkgdepR")` can be used to create interactive visualisations of 
+ dependencies between functions across packages.
+ - `r pkg("checked")`, `r pkg("prrd")`, and `r pkg("revdepcheck")` are designed 
+ to run reverse dependency checks in parallel. `usethis::use_revdep()` sets up 
+ a package to use `r pkg("revdepcheck")`.
```

Comment:
Although very useful revdepcheck status is a bit unclear, it had a deprecated notice for some time then it came back to be "active" in addition is only on github. I think it should be mention with more caution.

Reply:
Address in https://github.com/hturner/pkg-dev-ctv/pull/38/commits/610d15f4cb472fbd7972e437067974dad4c6364f